app/controllers/users/sessions_controller.rb

```diff
@@ -32,4 +32,11 @@ def after_sign_in_path_for(resource)
   def configure_sign_in_params
     devise_parameter_sanitizer.permit(:sign_in, keys: [:role])
   end
+  def only_senders
+    if current_user.role=='sender'
+      sender_dashboards_path
+    else
+      root_path
+    end 
+  end 
```

Comment:
why this method is in sessions controller? @aleemchishti

Reply:
deleted. only in sender dashboard now